### DISCUSSION TITLE
flashing end point

### DISCUSSION THREAD

**Discussion by sergejbuneev** [klinecharts/KLineChart#271] [General]
Good day! I'm trying to customize the end point and make light flashing animate with it. Could you please advice how to adjust it.
Below is a short video of how I expect to make its looks like

https://user-images.githubusercontent.com/99341029/178010656-4c84bff5-13ed-4043-b2a0-9adfeafd9e4d.mp4

**Comment by Emelya99** [comment]
It would also be interesting to know how to make a custom circle for priceMark.last instead of a line. How can this be implemented?
Thank you!)

**Reply by liihuu** [reply]
Yeah, that's a good idea.
